apps/web/components/LifeEventsCardsSection/components/LifeEventCard/LifeEventCard.treat.ts

```diff
@@ -23,4 +23,5 @@ export const card = style({
 
 export const content = style({
   flex: 1,
+  height: 156,
```

Comment:
We shouldn't be setting magic number heights

![image](https://user-images.githubusercontent.com/8494120/93119831-02cfb600-f6b2-11ea-97f3-19435081adcd.png)